src/Features/Core/Portable/TodoComments/ITodoCommentService.cs

```diff
@@ -43,18 +43,16 @@ public TodoComment(TodoCommentDescriptor descriptor, string message, int positio
             var originalLineInfo = location.GetLineSpan();
             var mappedLineInfo = location.GetMappedLineSpan();
 
-            return new TodoCommentData
-            {
-                Priority = Descriptor.Priority,
-                Message = Message,
-                DocumentId = document.Id,
-                OriginalLine = originalLineInfo.StartLinePosition.Line,
-                OriginalColumn = originalLineInfo.StartLinePosition.Character,
-                OriginalFilePath = document.FilePath,
-                MappedLine = mappedLineInfo.StartLinePosition.Line,
-                MappedColumn = mappedLineInfo.StartLinePosition.Character,
-                MappedFilePath = mappedLineInfo.GetMappedFilePathIfExist(),
-            };
+            return new(
```

Comment:
wary of this.  can we avoid using `new()` in cases where the type isn't immediately apparent?  From last design meeting we were ok with this being the default for fields, or locals with an explicit type.